test/e2e/e2e.go

```diff
@@ -1883,6 +1883,7 @@ var _ = ginkgo.Describe("e2e ingress traffic validation", func() {
 				}
 			}
 		})
+		// (tssurya): FIXME for LGW
```

Comment:
Is there a way by which I could read the test cluster's context and see if its localgateway and if yes, then skip the test?

Reply:
commenting it out for the moment, will fix it soon in an upcoming PR.